product_docs/docs/biganimal/release/using_cluster/05_monitoring_and_logging/monitoring_using_pemx/index.mdx

```diff
@@ -0,0 +1,93 @@
+
+### Time range picker
+
+On `Monitoring and Logging` tab, to view the data of a particular time range configure the time range using:
+- the `Last X` combo box
+- the date-time picker
+
+The `Last X` combo box provides several time range options in the drop-down list. Each option in the list is enabled only after that much time duration has elapsed since the PostgreSQL cluster is created. The default time range is of 15 minutes and the maximum is `Last 30 days`.
+
+!!! Note
+The data on the `Monitoring and Logging` tab is refreshed automatically every 30 seconds when the time range is selected from the `Last X` combo box.
+!!!
+
+### Information tooltip
+
+Each chart has a information tooltip that provides a detail description of the chart. Hover the mouse over the tooltip icon on the right side of the chart name to view the information.
+
+### Charts error state
+
+A red warning icon is displayed next to the tooltip if there is any error for that chart. If any of the metric exceeds its threshold it results into an error warning. Clicking on this red icon displays a window with description of the error and an `Edit Cluster` button. The `Edit Cluster` button takes to the edit cluster page. Do the configuration changes based on the specific metric that brings the cluster in a healthy state. The table shows list of different errors and the corresponding solutions.
+
+| Error  | Solution |
+| ------ | -------- |
+| **High CPU** | On the `Edit Cluster` page, select the preferred category, instance series, and the instance size to increase the CPU. |
+| **High Memory** | On the `Edit Cluster` page, select the preferred category, instance series, and the instance size to increase the memory size. |
+| **High Storage** | On the `Edit Cluster` page, go to cluster settings and increase the volume size to increase the storage. |
+
+## Features for historical charts
+
+You can [zoom in/out](#zooming-in/out-charts) the historical charts and also [download](#download-csv) the data of the historical charts.
+
+### Zooming in/out charts
+
+To zoom in a particular area on the historical chart, drag and select that specific area. To reset, select `Reset zoom` on the three dots icon at the top-right corner of the chart.
+
+### Download CSV
+
+To download the historical chart, go to the three dots icon in the top-right corner of the chart and select the `Download CSV` option from the drop-down list. The download includes only the data currently visible on the chart. To download the different data, configure the time range picker before selecting the `Download CSV` button.
```

Comment:
Regarding `To download the historical chart`, I think we need to make it more generic,
can we write something like, `This option allows the user to download the metrics data used to produce the chart in CSV format`

Reply:
changes done